analyze/aps-dnc.c

```diff
@@ -1447,6 +1447,17 @@ static void *get_edges(void *vaug_graph, void *node) {
 	    record_condition_dependencies(&f,cond,aug_graph);
 	    record_expression_dependencies(&f,cond,dependency,NO_MODIFIER,
 					   expr,aug_graph);
+
+            // Add edge from formal inside of case_expr to every Match inside Case 
+            Match m;
+            VERTEX sink;
+            sink.node = 0;
+            sink.modifier = NO_MODIFIER;
+            for (m = first_Match(some_case_stmt_matchers(case_stmt)); m;
+                m = MATCH_NEXT(m)) {
+              sink.attr = (Declaration)m;
```

Comment:
Casting a Match to a Declaration?

Reply:
I see that I do the same thing (see line 1575)

Reply:
But what is the purpose of saying that all the Match nodes depend on this formal?